R/plots.R

```diff
@@ -27,7 +27,15 @@ plotStudy <- function(study, modelID, featureID, plotID, testID = NULL, librarie
     is.null(libraries) || is.character(libraries)
   )
 
-  plots <- getPlots(study, modelID = modelID, libraries = libraries)
+  plots <- list()
+  for (i in seq_along(modelID)) {
+    tempPlots <- (getPlots(study, modelID = modelID[i], libraries = libraries))
+    if (i == 1) {
+      plots <- tempPlots
+    } else if (!is.null(tempPlots)) {
+      plots <- c(plots, tempPlots)
+    }
+  }
```

Comment:
I think you should be able to greatly simplify the above because adding `NULL` should vanish. For example:

```R
> x <- list()
> c(NULL, list(a = 1), NULL, list(b = 2), NULL, list(c = 3), NULL)
$a
[1] 1

$b
[1] 2

$c
[1] 3
```

Applying that to your loop above would look something like:

```R
  plots <- list()
  for (i in seq_along(modelID)) {
    tempPlots <- getPlots(study, modelID = modelID[i], libraries = libraries)
    plots <- c(plots, tempPlots)
  }
```

Reply:
Thanks for the comment. Implemented accordingly.